spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-seata/src/main/java/com/alibaba/cloud/seata/feign/SeataFeignObjectWrapper.java

```diff
@@ -37,21 +34,12 @@ public class SeataFeignObjectWrapper {
 
 	private final BeanFactory beanFactory;
 
-	private CachingSpringLoadBalancerFactory cachingSpringLoadBalancerFactory;
```

Comment:
Could you explain why you remove the code here and below?

Reply:
Ohoo ! , after removing the code for testing, I forgot to restore it .